src/side-nav.scss

```diff
@@ -72,6 +72,10 @@ $has-child-content-arrow-margin: 0.125rem;
   &--condensed {
     width: 2.75rem;
 
+    .#{$fd-namespace}-popover-custom {
+      display: block;
+    }
```

Comment:
Can't modify popover inside Side nav. This is against the rules for self-contained elements.

Reply:
created modifier class for this